modules/osdk-cli-reference-cleanup.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-cleanup_{context}"]
+= cleanup
+
+The `operator-sdk cleanup` command destroys and removes resources that were created for an Operator that was deployed with the `run` command.
+
+== packagemanifests
+
+`cleanup packagemanifests` subcommand destroys an Operator that was deployed with OLM by using the `run packagemanifests` command.
+
+.`packagemanifests` arguments
+[options="header",cols="1,3"]
+|===
+|Arguments |Description
+
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, Role, Subscription. These supplement or override defaults generated by run/cleanup.
```

Comment:
Please revise the first sentence to match a similar instance:

"The file path to Kubernetes resource manifests, such as `Role` objects and `Subscription` objects."

Consider revising the second sentence to be identical as well:

"These supplement or override defaults generated by `run` or `cleanup`."

I don't know the material, so I am unsure if the sentences should be identical or if there is an intentional difference.